src/ast/helpers.rs

```diff
@@ -729,6 +729,17 @@ pub fn is_docstring_stmt(stmt: &Stmt) -> bool {
     }
 }
 
+/// Return true if expr1 and expr2 are referring to same `Expr::Name` variable
+pub fn match_name_expr(expr1: &Expr, expr2: &Expr) -> bool {
```

Comment:
Check out `src/ast/comparable.rs`!

You can do:

```rs
let comparable_value: ComparableExpr = (&expr).into();
```

Then you can compare two expressions ignoring their locations. It's like this function, but generalized.

Reply:
Oh! great.